network/server_test.go

```diff
@@ -346,15 +347,18 @@ func TestNat(t *testing.T) {
 
 // TestPeerReconnection checks whether the node is able to reconnect with bootnodes on losing all active connections
 func TestPeerReconnection(t *testing.T) {
+	//Create bootnode config
```

Comment:
Nitpick warning: Please format the comment, add a space between // and the rest of the comment

Reply:
If anybody reading this, this is the commit that fixes it:

https://github.com/0xPolygon/polygon-sdk/pull/326/commits/d5b6d98bc3210e3a5cc730859d015ceb6d63857a